### PR TITLE
DOC-8283 PR #103786 - ui: add manual refresh option to the Active Executions in SQL Activity

### PR DESCRIPTION
**Pull request by florence-crl**
Addresses DOC-8283

(1) Updated Statements page and Transactions page, (a) added Refresh sub-section to Active Executions view section, (b) updated executions images.

Relevant rendered pages:
- [src/current/v23.2/ui-statements-page.md](https://deploy-preview-18114--cockroachdb-docs.netlify.app/docs/v23.2/ui-statements-page.html)
- [src/current/v23.2/ui-transactions-page.md](https://deploy-preview-18114--cockroachdb-docs.netlify.app/docs/v23.2/ui-transactions-page.html)

Note: the [Files changed comment](https://github.com/cockroachdb/docs/pull/18114#issuecomment-1836812846) incorrectly includes files that are NOT affected by this PR.

### CODE REVIEW

File: src/current/_includes/v23.2/ui/refresh.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Refresh
+
+{% include_cached new-in.html version="v23.2" %} To control refresh of the data on the Active Executions views of the SQL Activity Statements and Transactions pages, the following controls have been added
```
Comment (rmloveland) [inline src/current/_includes/v23.2/ui/refresh.md:3]: Copy editing comments:

- Add a colon after "following controls have been added"
- Capitalize all the sentences in the bulleted list (starting with "an","a","an")

File: src/current/_includes/v23.2/ui/refresh.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+- an **Active Statements|Transactions As Of** timestamp: Indicates when the last refresh was performed.
+- a manual **Refresh** button: When clicked, refreshes data immediately.
+- an **Auto Refresh** toggle: When toggled **On** (default), refreshes data immediately and then automatically every 10 seconds. When toggled **Off**, stops automatic data refresh. The toggle setting is shared by both the Statements and the Transactions pages. Changing the setting on one page changes it on the other page. 
+
+<img src="{{ 'images/v23.2/active-executions-view-refresh.png' | relative_url }}" alt="Active executions view refresh" style="border:1px solid #eee;max-width:100%" />
+
+If **Auto Refresh** is toggled **On**, navigating to the Active Executions view on either the Statements page or Transactions page, refreshes the data.
```
Comment (rmloveland) [inline src/current/_includes/v23.2/ui/refresh.md:11]: Copy edit: can remove the second comma in this sentence, e.g.

>  **Auto Refresh** is toggled **On**, navigating to the Active Executions view on either the Statements page or Transactions page refreshes the data.

**Review by rmloveland — APPROVED** [review]
LGTM! just had some copy edits

File: src/current/_includes/v23.2/ui/refresh.md (hunk 1 above)
Comment (florence-crl) [reply src/current/_includes/v23.2/ui/refresh.md:3]: updated

File: src/current/_includes/v23.2/ui/refresh.md (hunk 2 above)
Comment (florence-crl) [reply src/current/_includes/v23.2/ui/refresh.md:11]: updated

**Review by florence-crl — COMMENTED** [review]
TFTRs!

**Pull request merged**